WebRoot/skins/_base/base4/skin.properties

```diff
@@ -987,26 +987,26 @@ PropertySheetNoLeftSpacing  = padding-left:0px;
 ################
 #   SASH - splits a pane
 ################
-SashSize                    = 5px
-SashPadding                 = 1px
-Sash                        =
-SashContainer               = height:100%;
+SashSize                    = 1px
+SashPadding                 = 0px
```

Comment:
Those properties are already defined, I didn't created it though. And it just declares or set absolute value and not the css property, so it sounds okay to me. Let me know if you have any question.